specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,616 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview data policies REST API Document",
+    "version": "2023-08-01-preview"
+  },
+  "paths": {
+    "/data-policies": {
+      "get": {
+        "tags": [
+          "List_DataPolicy"
+        ],
+        "summary": "API to list the purview data policies",
+        "description": "List all data policies.",
+        "operationId": "List_DataPolicy",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy service",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        },
+        "x-ms-examples": {
+          "DevopsPolicy_List": {
+            "$ref": "./examples/DataPolicy_List.json"
+          }
+        }
+      }
+    },
+    "/data-policies/{policy-name}": {
+      "put": {
```

Comment:
Azure guidelines prefer PATCH over PUT for resource create -- it is more version resilient.

Reply:
In purview policy space we are currently supporting only the create and replace. Partial update of a policy is not supported.